overlays/moc/datacatalog/kustomization.yaml

```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - ../../../base/datacatalog
+
+patchesJson6902:
```

Comment:
Could you explain the naming here to me?

Reply:
Why is it called `patchesJson6902`?

Reply:
![image](https://user-images.githubusercontent.com/7453394/101032256-20f2d700-357a-11eb-9c2b-266efb4633bb.png)

Kustomize recognizes 2 patch strategies:
- [`patchesStrategicMerge`](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-api-machinery/strategic-merge-patch.md)
- [`patchesJson6902`](https://github.com/kubernetes-sigs/kustomize/blob/master/examples/jsonpatch.md)


It's derived from previous key, called `patches` (which used to be analogous to `patchesStrategicMerge` with some crazy additions). Which is kinda deprecated now and you should specify directly the strategy, since different plugin is being used. The naming comes with the same prefix `patches*`, so when the manifest's properties are ordered alphabetically, these are kept together.

Reply:
Thanks for that!

Reply:
@anishasthana see [here](https://github.com/kubernetes-sigs/kustomize/blob/master/examples/jsonpatch.md#json-patching) 
It's a different way of patching than the `strategicmergepatch` we usually do. It gives you more precision in where you want to patch, and exactly what fields to patch.